cli/.gitignore

```diff
@@ -2,3 +2,4 @@ artifact_info
 input.mender.txt
 test.txt
 testkey
+artifact.mender
```

Comment:
For future reference,

This and removing the test artifact (thank you for the cleanup!) should have been done on a separate commit.